Mathlib/RingTheory/Ideal/Operations.lean

```diff
@@ -1123,7 +1123,8 @@ In a Dedekind domain, to divide and contain are equivalent, see `Ideal.dvd_iff_l
 theorem le_of_dvd {I J : Ideal R} : I ∣ J → J ≤ I
   | ⟨_, h⟩ => h.symm ▸ le_trans mul_le_inf inf_le_left
 
-@[simp]
+/-- See also `isUnit_iff_eq_one`. -/
+@[simp high]
```

Comment:
This feels a bit like a hack to get around simpNF, is there another simp-lemma we can add to ensure the simp set is reasonable, in a different way?

Reply:
`Ideal.one_eq_top` already is simp. This is just to ensure that simp uses the direct lemma instead of two

Reply:
Okay, that's convincing.